app/src/main/java/com/tr4n/moviedb/ui/detail/MovieDetailFragment.kt

```diff
@@ -35,11 +48,34 @@ class MovieDetailFragment : BaseFragment(R.layout.fragment_movie_detail) {
     }
 
     override fun observeData() {
-        //TODO("Not yet implemented")
+        viewModel.movieDetail.observe(viewLifecycleOwner) {
+            movieDetail = it
+            Glide.with(this)
+                .load(Constant.BASE_URL_IMAGE + it?.backdropPath)
+                .into(imageViewBackdropPathMovieDetail)
+            Glide.with(this)
+                .load(Constant.BASE_URL_IMAGE + it?.posterPath)
+                .into(imageViewPosterPathMovieDetail)
+            val firstApiFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
+            val releaseDate = LocalDate.parse(it?.releaseDate, firstApiFormatter)
+            textReleaseYearMovieDetail.text = releaseDate.year.toString()
+            val runtime = it?.runtime.toString() + " Minutes"
+            textRuntimeMovieDetail.text = runtime
+            textOriginalTitle.text = it?.originalTitle
+            var movieGenres = ""
+            if (it?.genres != null) {
+                for (genre in it.genres) {
+                    movieGenres += genre.name + " "
+                }
+            }
+            textVoteAverage.text = it.voteAverage.toString()
+            textGenreMovieDetail.text = movieGenres
+            viewPageMovieDetail.layoutParams.height = activity?.container?.height!! - linearLayoutBottom.height
```

Comment:
không dùng !!
trong trường hợp cần thiết phải dùng elvis operator 
```suggestion
            viewPageMovieDetail.layoutParams.height = activity?.container?.height?:0 - linearLayoutBottom.height
```